src/Microsoft.DotNet.Interactive.Tests/VariableSharingTests.cs

```diff
@@ -542,7 +543,7 @@ await localCompositeKernel
                           remoteKernelUri);
 
             javascriptKernel.UseValueSharing();
-
+            javascriptKernel.RegisterCommandType<SendValue>();
```

Comment:
It shouldn't be necessary to register this on both the remote and the proxy. Is there a bug here?

Reply:
good point, not needed